workspaces/netlify-cms-widgets/src/youtube.tsx

```diff
@@ -86,8 +84,8 @@ function Control(props: CmsWidgetControlProps & any) {
       >
         {fetching
           ? `Fetching: ${fetching}`
-          : title
-          ? `Video: ${title}`
+          : title && formattedDuration
+          ? `Video: ${title} | Duration: ${formattedDuration}`
```

Comment:
`formattedDuration` is never defined in this file

Reply:
You are right. Thanks, Tsotne, removed.